### DISCUSSION TITLE
AES Key

### DISCUSSION THREAD

**Discussion by Amorph** [X1Plus/X1Plus#115] [Q&A]
Hello everyone,
I want to try decrypt P1 firmware with same algo like in for X1 but I need UPDATE_KEY_MATERIAL which used for decryption.
As I understand, it's one for same model and I hope it's same between models, and I want to try

**Comment by jwise** [comment]
Unfortunately, we can't allow discussion of swapping AES keys in this repo :( You'll need an X1 in order to get the X1 key.  (Though we believe that the key is different for P1, anyway.)

**Reply by Amorph** [reply]
You are fast, okay, got it)
